### ISSUE TITLE
Ulduar  Steelforged defenders

### ISSUE THREAD

**Issue by knifeofbleed** [dalaranwow/dalaran-wow#5886]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: to count the achievment when you kill them didnt count

**Steps to reproduce the problem**:

1. when you kill the dwarfes achievment didnt count
2. also about when storm beacons are down they didnt close the portal soo they keep summoning them even beacons is destroyed
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (steelforged defenders ), spells, items, quests, instances, zones,  [Dwarfageddon (10 player)]  [Dwarfageddon (25 player)], skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if neces 
https://prnt.sc/sa205o for beacons they are destroyed but keep summoning them
https://prnt.sc/sa20ri you kill them but didnt count on the achievment even secounds when you kill 1st https://prnt.sc/sa20wz
